hack/verify-flags/known-flags.txt

```diff
@@ -72,6 +72,8 @@ e2e-output-dir
 enable-debugging-handlers
 enable-horizontal-pod-autoscaler
 enable-deployment-controller
+enable-hostpath-provisioner
```

Comment:
Each time I see an artifact of the hostpath provision knob I am more convinced it should go away :)